aerogear/aerogear-ios-sdk
# added security module with checks

### Motivation

JIRAS: [1](https://issues.jboss.org/browse/AEROGEAR-{2124}), [2](https://issues.jboss.org/browse/AEROGEAR-{2125}), [3](https://issues.jboss.org/browse/AEROGEAR-{2126})


### Description

Add individual device checks as per the JIRAs listed above.

### Progress

- [x] Jailbreak
- [x] Device lock
- [x] Debugger check 

### Verification

This PR can be verified together with [this one](https://github.com/aerogear/ios-showcase-template/pull/19).

* Clone this branch
* Use the included scripts to clone the showcase application to within the sdk.
* Check out the linked branch of the showcase app above
* Run the showcase application and verify that all device trust checks perform as expected
